src/network/dandelion.py

```diff
@@ -6,10 +6,10 @@
 from random import choice, expovariate, sample
 from threading import RLock
 from time import time
-
-import connectionpool
-import state
-from queues import invQueue
+try:
+    from queues import invQueue
+except ImportError:
+    from pybitmessage.queues import invQueue
```

Comment:
I would also pass `invQueue` using dependency injection. Then we can simplify import.

Reply:
At Constructor level, dependency injection is not working, having same issue. Dandelion class have two fns having invQueue dependency. `from pybitmessage.queues import invQueue` is working but for portable it fails.

Reply:
i think try except option would be better here until project fully migrated to py3

Reply:
Used dependency injection at function level.